articles/150_ros_command_line_arguments.md

```diff
@@ -201,6 +202,25 @@ ros2 run some_package some_ros_executable --ros-args --disable-rosout-logs --dis
 
 Logging is fully enabled by default, thus `--enable-*` options are usually redundant unless a `--disable-*` option found earlier in the command line is being overridden.
 
+#### Enclave assignments
+
+Enclave assignment may be achieved using the `--enclave`/`-e` option.
+This option takes a single string `value` assignment statement, where `value` is a fully qualified enclave path used to locate the respective security artifacts within the configured keystore.
+
+As an example, to assign an enclave path `/foo/bar` one may execute:
+
+```sh
+ros2 run some_package some_ros_executable --ros-args --enclave="/foo/bar"
```

Comment:
I slightly prefer to be more explicit and include `security` in the long flag version: `--security-enclave`.
The short version is fine.

Reply:
as we sided for `enclave` (without security) all throughout the stack, changing just the flag will introduce inconsitency both between short/long flag but also with the rest of the functions, variables, XML tag names etc. So if it is changed IMO it should be changed everywhere.

Picking an unambiguous name "enclave" as opposed to ovreloaded terms like "context" allows us to coin that term for security purposes.
So my preference goes for not changing it anywhere.

Reply:
sounds good to me